config/pitest-suppressions/pitest-indentation-suppressions.xml

```diff
@@ -126,6 +126,15 @@
     <lineContent>primordialHandler.checkIndentation();</lineContent>
   </mutation>
 
+  <mutation unstable="false">
```

Comment:
We will not allow extension of pitest suppressions for this update, we need to find a test case to kill this mutation or modify the existing code

Reply:
Please try to cover, if you feel you are stuck, consider https://github.com/checkstyle/checkstyle/wiki/How-to-Generate-and-Analyze-Pitest-Reports

If still stuck, let us know we will assist.